kolibri/utils/kolibri_js_names.json

```diff
@@ -0,0 +1,4 @@
+{
+    "KOLIBRI_CORE_JS_NAME": "kolibriGlobal",
+    "KOLIBRI_JS_PLUGIN_DATA_NAME": "__kolibriPluginData"
+}
```

Comment:
Since there are so many variable names for these strings, it ironically makes it harder to figure out what the actual global variable is actually called: `KOLIBRI_JS_PLUGIN_DATA_NAME`, `__kolibriPluginData`, `kolibriPluginDataName`, `__kolibriPluginDataName`... but all the dev wants to do is inspect it in their console.

I think in this case, it would be preferable to lower the amount of indirection and allow the strings to be defined more than once in the codebase. If the strings are unique, meaningful, and unlikely to change, there's not a huge win from having a single source of truth. So for example:

* `window.theKolibriPluginData`
* `window.theKolibriCoreApp`

These are unique, easily-searched strings and use the [singleton naming convention](https://vuejs.org/v2/style-guide/#Single-instance-component-names-strongly-recommended). Also since they're named similarly, they'll be clustered and easy to find in the dev tools.

Reply:
Yeah - as I was doing the extension here I was having a similar thought. Especially as it just moves the 'shared nameness' to the constant variable name, rather than the value itself.

I can clean this up in a follow up.